### PR TITLE
Add machine files for the Livermore Computing clusters quartz and syrah

### PR DESCRIPTION
**Pull request by AaronDonahue**
This commit will allow us to set up nightly testing on the livermore computing systems
quartz and syrah.

Additionally, it will provide a template for building scream on these machines.

### CODE REVIEW

File: components/scream/cmake/machine-files/quartz.cmake [hunk 1]
```diff
@@ -0,0 +1,7 @@
+set(CMAKE_CXX_FLAGS "-w" CACHE STRING "")
+set(KOKKOS_ENABLE_DEBUG FALSE CACHE BOOL "")
+set(KOKKOS_ENABLE_AGGRESSIVE_VECTORIZATION FALSE CACHE BOOL "")
+set(KOKKOS_ENABLE_CUDA FALSE CACHE BOOL "")
```
Comment (jgfouca) [inline components/scream/cmake/machine-files/quartz.cmake]: If quarts and syrah are openMP machines, you should not use the CUDA machine .cmake files as a template. You can use blake as a template but be sure to get the architecture right.

Comment (AaronDonahue) [reply components/scream/cmake/machine-files/quartz.cmake]: this latest commit should fix this, can you take a look?

**Comment by AaronDonahue** [comment]
@jgfouca , I believe it does.  I followed the guidance here: https://github.com/kokkos/kokkos/wiki/Compiling.  And yes BDW stands for Broadwell, quartz and syrah both have Intel Xeon E5-2695 v4 chips which I believe are Broadwell.  Correct me if I'm wrong though.

**Comment by jgfouca** [comment]
Great, feel free to merge.

**Pull request merged**